example/json.html

```diff
@@ -0,0 +1,251 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>flowchart.js · Playground</title>
+    <style type="text/css">
+        .end-element { background-color : #FFCCFF; }
+    </style>
+    <script src="http://cdnjs.cloudflare.com/ajax/libs/raphael/2.1.0/raphael-min.js"></script>
+    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
+    <!--<script src="http://flowchart.js.org/flowchart-latest.js"></script>-->
+   <script src="../release/flowchart.js"></script>
+    <script>
+
+        window.onload = function () {
+            var btn = document.getElementById("run"),
+                    cd = document.getElementById("code"),
+                    chart;
+
+            (btn.onclick = function () {
+                var code = cd.value;
+
+                if (chart) {
+                    chart.clean();
+                }
+
+
+
+                var data = JSON.parse(code);
+
+                var symbols=data.symbols;
+                var directions=data.directions;
+
+
+                var symbolsArray ={};
+                for(var key in symbols)
+                {
```

Comment:
Are we going with this C style curly braces?

Reply:
Bad idea in JavaScript. Every newLine triggers a “Should a ; be auto inserted. In compiled languages like C and Java the position of the { is a question of taste (and a source of religious wars second only to tabs vs spaces). In JavaScript it matters. Keep them at line end.